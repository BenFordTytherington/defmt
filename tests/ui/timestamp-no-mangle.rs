#![no_main]
#![no_std]

#[defmt::timestamp]
#[no_mangle]
fn foo() -> u64 {
    0
}
