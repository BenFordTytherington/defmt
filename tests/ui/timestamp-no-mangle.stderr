error: `#[timestamp]` attribute cannot be used together with `#[no_mangle]`
 --> $DIR/timestamp-no-mangle.rs:5:1
  |
5 | #[no_mangle]
  | ^^^^^^^^^^^^
